components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/DefaultSessionManager.cs

```diff
@@ -142,7 +142,7 @@ public class DefaultSessionManager : ISessionManager, IDisposable {
                     _sessions.Add(id, wrapper);
                 }
                 wrapper._subscriptions.AddOrUpdate(subscription.Id, subscription);
-                _logger.Information("Subscription '{id}' registered in session '{id}' in state {state}",
+                _logger.Verbose("Subscription {subscriptionId} registered in session {id} in state {state}",
```

Comment:
>Verbose [](start = 24, length = 7)

Typically this should only happen once during runtime. I think is a good idea to keep it as information so that we can follow the flow in a debugging case.

Reply:
5530 hits in 10 min...